src/test/java/org/eclipse/yasson/Jsonbs.java

```diff
@@ -0,0 +1,13 @@
+package org.eclipse.yasson;
+
+import javax.json.bind.*;
+import org.eclipse.yasson.internal.*;
+
+public class Jsonbs {

```

Comment:
this class is fine for now, but I will probably undo this in a later PR for when I implement #333

Reply:
I see, if you're gonna do this sometime in the future then yes.. 'Jsonbs' is pointless.